bigtable/README.MD

```diff
@@ -0,0 +1,46 @@
+# Google Cloud Bigtable via Apache HBase
+This initialization action installs Apache HBase libraries and the [Google Cloud Bigtable](https://cloud.google.com/bigtable/) [HBase Client](https://github.com/GoogleCloudPlatform/cloud-bigtable-client).
+
+
+## Using this initialization action
+You can use this initialization action to create a Dataproc cluster configured to connect to Cloud Bigtable:
+
+1. Create a Bigtable instance by following [these directions](https://cloud.google.com/bigtable/docs/creating-instance).
+1. Using the `gcloud` command to create a new cluster with this initialization action.
+
+    ```bash
+    gcloud dataproc clusters create <CLUSTER_NAME> \
+    --initialization-actions gs://dataproc-initialization-actions/bigtable/bigtable.sh \
+    --metadata bigtable-instance=<BIGTABLE INSTANCE>
+    ```
+1. When it is finished being created the cluster will have HBase libraries and the Bigtable Client.
+1. You can validate the deployment by SSHing to the master (`gcloud compute ssh <CLUSTER_NAME>-m`) and running `hbase shell`.
+
+## Running example MR job on cluster
+1. Get the code:
+   ```bash
+   git clone https://github.com/GoogleCloudPlatform/cloud-bigtable-examples/tree/master/java/dataproc-wordcount
+   ```
+1. Compile:
+    ```bash
+    mvn clean package -Dbigtable.projectID=<BIGTABLE PROJECT> -Dbigtable.instanceID=<BIGTABLE INSTANCE>
+    ```
+1. The job will create two jars - with and without dependencies included.
+1. Use the jar with dependecies to submit job:
+   
+    ```bash
+    gcloud dataproc jobs submit hadoop --cluster <CLUSTER_NAME> --class com.example.bigtable.sample.WordCountDriver --jars target/wordcount-mapreduce-0-SNAPSHOT-jar-with-dependencies.jar -- wordcount-hbase gs://dataproc-initialization-actions/README.md <HBASE_TABLE>
+    ```
+
+## Running example Spark job on cluster
+1. On the cluster there is Spark-HBase connector installed. Please find more details regarding compilation and example usage on [Apache Spark - Apache HBase Connector](https://github.com/hortonworks-spark/shc)
```

Comment:
Nit (replacing the first sentence): "This initialization action adds the Spark-HBase connector to the Spark classpath"